test/release/examples/primers/domains.chpl

```diff
@@ -30,49 +30,53 @@
 config var n = 10;
 
 var RD: domain(3) = {1..n, 1..n, 1..n};
-writeln(RD);
+writeln(RD); // {1..10, 1..10, 1..10}
 
 //
 // Rectangular domains have a set of methods that enable convenient
 // reuse of existing domains.
 //
 // The :proc:`~ChapelDomain.expand` method returns a new domain that is expanded or
-// contracted depending on the sign of the offset argument.
+// contracted in each dimension depending on the value and sign of the offset
+// argument.  The expansion/contraction in each dimension happens symmetrically to
+// the lower and upper bound.
 //
-var RDbigger = RD.expand((1,1,1));
-writeln(RDbigger);
-var RDsmaller = RD.expand((-1,-1,-1));
-writeln(RDsmaller);
+var RDexp = RD.expand((1,2,-2));
+writeln(RDexp); // {0..11, -1..12, 3..8}
 
 //
-// The :proc:`~ChapelDomain.exterior` method returns a new domain that is the
-// exterior portion of the current domain.  A positive offset specifies that
-// the exterior should be taken from the high bound; a negative offset, the low
-// bound.
+// The :proc:`~ChapelDomain.exterior` method returns a new domain that is an
+// exterior portion of the current domain.  This is not the "border" of the
+// domain (which is not a rectangular domain), in particular the new domain
+// will share no indices with the original domain unless an offset is 0, in which
+// case the respective range will be that of the original domain.
+// Inputs specify the width of the new domain in the respective dimension.
+// A positive value specifies that the exterior should be taken from the high
+// bound; a negative offset, the low bound.
 //
-var RDext_p = RD.exterior((1,1,1));
-writeln(RDext_p);
-var RDext_n = RD.exterior((-1,-1,-1));
-writeln(RDext_n);
+var RDext0 = RD.exterior((1,4,-4));
+writeln(RDext0); // {11..11, 11..14, -3..0}
+var RDext1 = RD.exterior((0,4,0));
+writeln(RDext1); // {1..10, 11..14, 1..10}
 
 //
 // The :proc:`~ChapelDomain.interior` method returns a new domain that is the
-// interior portion of the current domain.  The sign of the offset implies
+// interior portion of the current domain.  Inputs specify the width
+// of the new domain in the respective dimension.  The sign of the offset implies
 // using the high or low bound as in the exterior case.
 //
-var RDint_p = RD.interior((1,1,1));
-writeln(RDint_p);
-var RDint_n = RD.interior((-1,-1,-1));
-writeln(RDint_n);
+var RDint = RD.interior((2,-5,1));
+writeln(RDint); // {9..10, 1..5, 10..10}
 
 //
 // The :proc:`~ChapelDomain.translate` method returns a new domain that is the
-// current domain translated by the offset.
+// current domain translated by the offset. If only a single offset is passed
+// it will apply to all dimensions.
 //
-var RDtrans_p = RD.translate((1,1,1));
-writeln(RDtrans_p);
-var RDtrans_n = RD.translate((-1,-1,-1));
-writeln(RDtrans_n);
+var RDtrans0 = RD.translate((0,4,-4));
+writeln(RDtrans0); // {1..10, 5..14, -3..6}
+var RDtrans1 = RD.translate(4);
+writeln(RDtrans1); // {5..14, 5..14, 5..14}
 
```

Comment:
For the "single offset" note, I was envisioning that it would be in its own paragraph and indicate that it applies to all four methods. For example, place the sentence "If only a single offset is passed it will apply to all dimensions." between the two RDtrans variables:

```chpl
.....
writeln(RDtrans1);   // renamed from RDtrans0

//
// The methods expand, exterior, interior, and translate
// can also accept a single offset. If so, this offset
// is applied to all dimensions of the domain.
//
var RDtrans2 = RD.translate(4);
writeln(RDtrans2); // {5..14, 5..14, 5..14}
```

Reply:
Done!